README.md

```diff
@@ -21,8 +21,7 @@ with Tableau's line of products. It is based on the [PostgreSQL JDBC driver].
 
 ## Install
 
-You will need to acquire and install two software artefacts, the PostgreSQL
-JDBC driver ([JAR file]), and the CrateDB connector ([TACO file]).
+You will need to download the PostgreSQL JDBC driver ([JAR file]), and the CrateDB connector ([TACO file]).
```

Comment:
Should we say that latest version of the PG JDBC driver doesn't work with CrateDB?

Reply:
Its already mentioned a few lines below but I could see myself ignoring that and jumping right into the download page!

I'll move it upwards

Reply:
Also removed mentioning CrateDB jdbc as its currently something we do not recommend, mentioning it could be confusing for the user.